ui/pages/confirmations/hooks/alerts/useBlockaidAlerts.test.ts

```diff
@@ -78,6 +78,8 @@ describe('useBlockaidAlerts', () => {
     });
 
     expect(result.current).toHaveLength(1);
+    expect(result.current[0].reportUrl).toBeDefined();
+    delete result.current[0].reportUrl;
```

Comment:
why not add the `reportUrl` to the `EXPECTED_ALERT`?

Reply:
`reportUrl` generated are different on local and CI I think due to zip utility and that breaks the unit test.

Reply:
could we add this as a comment where we delete reportUrl?